pyproject.toml

```diff
@@ -18,3 +18,8 @@ extend-exclude = '''
   | dist
 )/
 '''
+
+[tool.isort]
+profile = "black"
+reverse_relative = true
```

Comment:
Does this sort more deep (i.e. more dots) first?
@CodyCBakerPhD

Reply:
I _think_ it's the other way, more local first